recipes-extended/libcobalt/libcobalt_git.bb

```diff
@@ -53,45 +58,33 @@ def get_cobalt_data_path(d):
         return "${WPEFRAMEWORK_DATA_PATH}/Cobalt"
 COBALT_DATA = "${@get_cobalt_data_path(d)}"
 
-do_configure() {
-    export PATH="$PATH:${S}/depot_tools"
+do_compile() {
```

Comment:
Please split configure from the build here. On the build level, only ninja is necessary to execute. GN is needed only to generate the build configuration

Reply:
done